examples/common/nanovg/nanovg_bgfx.cpp

```diff
@@ -683,38 +692,51 @@ namespace
 		}
 	}
 
-	static const uint64_t s_blend[] =
-	{
-		BGFX_STATE_BLEND_ZERO,
-		BGFX_STATE_BLEND_ONE,
-		BGFX_STATE_BLEND_SRC_COLOR,
-		BGFX_STATE_BLEND_INV_SRC_COLOR,
-		BGFX_STATE_BLEND_DST_COLOR,
-		BGFX_STATE_BLEND_INV_DST_COLOR,
-		BGFX_STATE_BLEND_SRC_ALPHA,
-		BGFX_STATE_BLEND_INV_SRC_ALPHA,
-		BGFX_STATE_BLEND_DST_ALPHA,
-		BGFX_STATE_BLEND_INV_DST_ALPHA,
-		BGFX_STATE_BLEND_SRC_ALPHA_SAT,
-	};
-
 	static uint64_t glnvg_convertBlendFuncFactor(int factor)
 	{
-		const uint32_t numtz = bx::uint32_cnttz(factor);
-		const uint32_t idx   = bx::uint32_min(numtz, BX_COUNTOF(s_blend)-1);
-		return s_blend[idx];
+		if (factor == NVG_ZERO)
```

Comment:
NVGblendFactor is not defined as 1, 2, 3, ... in the `nanovg.h`. Index should not work as expected.

```enum NVGblendFactor {
	NVG_ZERO = 1<<0,
	NVG_ONE = 1<<1,
	NVG_SRC_COLOR = 1<<2,
	NVG_ONE_MINUS_SRC_COLOR = 1<<3,
	NVG_DST_COLOR = 1<<4,
	NVG_ONE_MINUS_DST_COLOR = 1<<5,
	NVG_SRC_ALPHA = 1<<6,
	NVG_ONE_MINUS_SRC_ALPHA = 1<<7,
	NVG_DST_ALPHA = 1<<8,
	NVG_ONE_MINUS_DST_ALPHA = 1<<9,
	NVG_SRC_ALPHA_SATURATE = 1<<10,
};